other/xml.sh

```diff
@@ -2,23 +2,23 @@
 
 SVGDIR="../icons/"
 EXPORT="../app/src/main/res"
-ICPACK_PRE='        <item>'
+ICPACK_PRE='	    <item>'
```

Comment:
I wanted to avoid tabs, what are the whitespace changes here?

Reply:
Oh, I dunno a contributor changed it I think, you also just can add  `-V` behind sort haha.